tests/test_creation/test_mathematical_combination.py

```diff
@@ -100,7 +100,7 @@ def test_default_parameters(df_vartypes):
             "min(Age-Marks)": [0.9, 0.8, 0.7, 0.6],
         }
     )
-
+    ref['City'] = ref['City'].astype("category")
     # init params
```

Comment:
same here, this test does not operate over categorical variables. I would try not to modify this test.